README.md

```diff
@@ -47,7 +47,9 @@ Here you will find examples how to use Amazon Forecast Python SDK to make API ca
   - [import data containing 1 single item](https://github.com/aws-samples/amazon-forecast-samples/blob/master/notebooks/basic/Tutorial/1.Importing_Your_Data.ipynb)
   - [train a predictor and forecast on a single item](https://github.com/aws-samples/amazon-forecast-samples/blob/master/notebooks/basic/Tutorial/2.Building_Your_Predictor.ipynb)
   - [query and evaluate one item at a time](https://github.com/aws-samples/amazon-forecast-samples/blob/master/notebooks/basic/Tutorial/3.Evaluating_Your_Predictor.ipynb)
+
 - Advanced folder contains notebooks to show API calls for more complex tasks:
+  - [Time Series Clustering Preprocessing](https://github.com/aws-samples/amazon-forecast-samples/blob/master/notebooks/advanced/Clustering_Preprocessing/02.%20Time%20Series%20Clustering%20Using%20DTW%20KMeans.ipynb)
```

Comment:
May be better to link to README file, because it contains the overview information and links to 2 notebooks.

Reply:
Nice suggestion, updated